android/src/main/java/com/globo/reactnativeua/ReactNativeUA.java

```diff
@@ -81,5 +101,7 @@ public void disableActionUrl() {
 
     @ReactMethod
     public void enableGeolocation() {
```

Comment:
It`s done this way in order to avoid impact in the device battery. But since the default behavior is background disabled we could just remove this line.